### PR TITLE
Add Aerospike DB

### PR DESCRIPTION
**Pull request by bruwozniak**

### CODE REVIEW

File: cmd/go-ycsb/client.go [hunk 1]
```diff
@@ -17,16 +17,15 @@ import (
...
 	"time"
 
+	"github.com/magiconair/properties"
 	"github.com/pingcap/go-ycsb/pkg/measurement"
 	"github.com/pingcap/go-ycsb/pkg/prop"
 	"github.com/spf13/cobra"
-
-	"github.com/magiconair/properties"
-	"os"
 )
```
Comment (kostyantyn) [inline cmd/go-ycsb/client.go:29]: 😍

**Review by kostyantyn — APPROVED** [review]

File: cmd/go-ycsb/main.go [hunk 2]
```diff
@@ -42,10 +42,14 @@ import (
 	_ "github.com/pingcap/go-ycsb/db/mysql"
 	// Register TiKV database
 	_ "github.com/pingcap/go-ycsb/db/tikv"
+	// Register PostgreSQL database
+	_ "github.com/pingcap/go-ycsb/db/pg"
```
Comment (miry) [inline cmd/go-ycsb/main.go:46]: @bruwozniak should it be in separate PR?

**Comment by miry** [comment]
@bruwozniak No more Aerospike DB?
